pynecone/app.py

```diff
@@ -324,35 +333,39 @@ async def _ping() -> str:
     return "pong"
 
 
-def _event(app: App) -> Reducer:
-    """Create an event reducer to modify the state.
-
-    Args:
-        app: The app to modify the state of.
+def _event(app: App):
+    async def ws(websocket: WebSocket):
+        await websocket.accept()
+        while True:
+            data = await websocket.receive_text()
+            data = data.replace("True", "true").replace("False", "false")
```

Comment:
I need to remove this line, something wasn't getting converted to JSON correctly along the way